node/repo/fsrepo.go

```diff
@@ -50,6 +50,8 @@ func NewRepoTypeFromString(t string) RepoType {
 		return Worker
 	case "Wallet":
 		return Wallet
+	case "Follower":
+		return Follower
```

Comment:
This shouldn't be a new node type, just use FullNode here (really that name should be "ChainNode", but let's not refactor code in this PR)

Reply:
resolved #3d3d2891c

Reply:
resolved #3d3d2891c